person.rb

```diff
@@ -1,4 +1,5 @@
 require './corrector'
+require './rental'
 
 class Person
```

Comment:
- [ ] A method for adding rentals should be present in the Person class and the person itself should be sent as a parameter to create the new rental achieving the `has-many` association. The piece of code below may be helpful to you.

```ruby
def add_rental(book, date)
      Rental.new(date, book, self)
end
```